network.c

```diff
@@ -14,7 +14,8 @@ struct network_box_result perform_network_detect(network *n, image *img, int cla
         sized = resize_image(*img, n->w, n->h);
     }
     struct network_box_result result = { NULL };
-    network_predict(*n, sized.data);
+    // mleak at network_predict(), get_network_boxes() and network_predict_ptr()?
+    network_predict_ptr(n, sized.data);
```

Comment:
It would be great if it has a binding to `free_network_ptr` as well. What do you think?

Reply:
We already have one, but it works with no-pointer:
```go
C.free_network(*n.cNet)
```

switched to pointer version:
```go
C.free_network_ptr(n.cNet)
```
Link:
https://github.com/LdDl/go-darknet/pull/16/files#diff-bdcd879439d041827d334846f9aba01de6e3683ed8fdd01e63917dae6df23846L61

Or do you think it would be better to have both Close() and ClosePtr()?

Reply:
If it works for both, it's okay to me. 🚀